promql/generated_parser.y

```diff
@@ -35,120 +36,351 @@ import (
     series    []sequenceValue
     uint      uint64
     float     float64
+    string    string
+    duration  time.Duration
 }
 
 
-%token <item> ERROR 
-%token <item> EOF
```

Comment:
I'd leave them like this, it makes history of source control easier.

Reply:
How about this form:
```
%token <item>
TOKEN1
TOKEN2
...
```
This is what I've done now.